pages/api/check-status.ts

```diff
@@ -8,14 +7,17 @@ import {
   CheckStatusApiRequestQuery,
 } from '../../lib/types'
 import passportStatusesMock from '../../__mocks__/passportStatusesMock.json'
+import { getLogger } from '../../logging/log-util'
+
+const logger = getLogger('check-status')
 
 export default async function handler(
   req: NextApiRequest,
   res: NextApiResponse<CheckStatusApiResponse | string>
 ) {
   if (req.method !== 'GET') {
-    res.status(405).send(`Invalid request method ${req.method}`)
-    return
+    logger.error(`error 405: Invalid request method ${req.method}`)
```

Comment:
I would log this as either `warn` or `debug` because this isn't an application error.